src/index.js

```diff
@@ -1,14 +1 @@
-import _ from 'lodash';
 import './style.css';
```

Comment:
Great work @limavfabio 🤝

- [ ] Since this project requires to use of Javascript, kindly add some javascript code to manipulate the DOM. Instead of having all DOM code in the HTML file, you may add some function that you may `export` as a `module` just as required in the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md).